dashboard/src/app.jsx

```diff
@@ -102,24 +102,47 @@ const App = ({ resetRecoil }) => {
 
   useEffect(() => {
     const onWindowFocus = (e) => {
-      if (authToken && e.newState === "active") {
-        API.get({ path: "/check-auth" }) // will force logout if session is expired
+      if (api.getToken() && e.newState === "active") {
+        api
+          .get("/check-auth") // will force logout if session is expired
           .then(() => {
             if (initialLoadIsDone) {
               // if the app is already loaded
               // will refresh data if session is still valid
               refresh();
-            } else {
-              console.log("initial load not done");
             }
+          })
+          .catch(() => {
+            toast.error("Impossible de vérifier votre session. Vous avez été déconnecté.");
```

Comment:
1. le comportement actuel c'est `window.location.reload();` qui fait qu'on n'affiche de toute façon pas de `toast.error` donc est-ce nécessaire ici ?
2. si c'est nécessaire ici, je mettrais plutôt un autre message
```suggestion
            toast.error("Votre session est expirée, veuillez vous reconnecter.");
```